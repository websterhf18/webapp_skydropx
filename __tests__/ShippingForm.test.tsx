/**
 * @format
 * @jest-environment jsdom
 */
import React from "react";
import renderer from "react-test-renderer"; // Note: test renderer must be required after react-native.
import ShippingForm from "../components/Forms/Shipping.form";

it("Renders the Shipping Form Component", () => {
  const tree = renderer
    .create(
      <ShippingForm
        onSubmit={() => {
          console.log("submitting shipping form");
        }}
      />,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
