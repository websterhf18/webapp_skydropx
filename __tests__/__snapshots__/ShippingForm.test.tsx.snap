// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders the Shipping Form Component 1`] = `
<form
  className="needs-validation m-3"
  onSubmit={[Function]}
>
  <div
    className="row g-3"
  >
    <div
      className="col-sm-6"
    >
      <label
        className="form-label"
        htmlFor="firstName"
      >
        Codigo postal origen
      </label>
      <input
        className="form-control "
        data-testid="codigo_origen_test"
        name="codigo_origen"
        onBlur={[Function]}
        onChange={[Function]}
        type="text"
        value=""
      />
    </div>
    <div
      className="col-sm-6"
    >
      <label
        className="form-label"
        htmlFor="lastName"
      >
        Codigo postal destino
      </label>
      <input
        className="form-control "
        data-testid="codigo_destino_test"
        name="codigo_destino"
        onBlur={[Function]}
        onChange={[Function]}
        type="text"
        value=""
      />
    </div>
    <div
      className="col-md-3"
    >
      <label
        className="form-label"
        htmlFor="lastName"
      >
        Peso
      </label>
      <input
        className="form-control "
        data-testid="peso_test"
        name="peso"
        onBlur={[Function]}
        onChange={[Function]}
        type="number"
      />
    </div>
    <div
      className="col-md-3"
    >
      <label
        className="form-label"
        htmlFor="lastName"
      >
        Altura
      </label>
      <input
        className="form-control "
        data-testid="altura_test"
        name="altura"
        onBlur={[Function]}
        onChange={[Function]}
        type="number"
      />
    </div>
    <div
      className="col-md-3"
    >
      <label
        className="form-label"
        htmlFor="lastName"
      >
        Ancho
      </label>
      <input
        className="form-control "
        data-testid="ancho_test"
        name="ancho"
        onBlur={[Function]}
        onChange={[Function]}
        type="number"
      />
    </div>
    <div
      className="col-md-3"
    >
      <label
        className="form-label"
        htmlFor="lastName"
      >
        Largo
      </label>
      <input
        className="form-control "
        data-testid="largo_test"
        name="largo"
        onBlur={[Function]}
        onChange={[Function]}
        type="number"
      />
    </div>
  </div>
  <hr
    className="my-4"
  />
  <button
    className="w-100 btn btn-primary btn-lg"
    data-testid="send_submit_test"
    disabled={false}
    type="submit"
  >
    Enviar
  </button>
</form>
`;
